docs/release-notes/release-notes-1.7.0.md

```diff
@@ -0,0 +1,1434 @@
+  versions
+* Further embrace the increased security, and other desirable properties, that
+  hard forks provide over soft forks
+
+See the following for more details:
+
+* [Politeia proposal](https://proposals.decred.org/record/3a98861)
+* [DCP0008](https://github.com/decred/dcps/blob/master/dcp-0008/dcp-0008.mediawiki)
+
+### Automatic Ticket Revocations Vote
+
+The third new vote available as of this release has the id `autorevocations`.
+
+The primary goals of this change are to:
+
+* Improve the Decred stakeholder user experience by removing the requirement for
+  stakeholders to manually revoke missed and expired tickets
+* Enable the recovery of funds for users who lost their redeem script for the
+  legacy VSP system (before the release of vspd, which removed the need for the
+  redeem script)
+
+See the following for more details:
+
+* [Politeia proposal](https://proposals.decred.org/record/e2d7b7d)
+* [DCP0009](https://github.com/decred/dcps/blob/master/dcp-0009/dcp-0009.mediawiki)
+
+### Change PoW/PoS Subsidy Split to 10/80 Vote
+
+The fourth new vote available as of this release has the id `changesubsidysplit`.
+
+The proposed modification to the subsidy split is intended to substantially
+diminish the ability to attack Decred's markets with mined coins and improve
+decentralization of the issuance process. 
+
+See the following for more details:
+
+* [Politeia proposal](https://proposals.decred.org/record/427e1d4)
+* [DCP0010](https://github.com/decred/dcps/blob/master/dcp-0010/dcp-0010.mediawiki)
+
+### Unspent Transaction Output Overhaul
+
+The way unspent transaction outputs (UTXOs) are handled has been significantly
+reworked to provide major performance enhancements to both steady-state
+operation as well as the initial chain sync process as follows:
+
+* Each UTXO is now tracked independently on a per-output basis
+* The UTXOs now reside in a dedicated database
+* All UTXO reads and writes now make use of a cache
+
+#### Unspent Transaction Output Cache
+
+All reads and write of unspent transaction outputs (utxos) now go through a
+cache that sits on top of the utxo set database which drastically reduces the
+amount of reading and writing to disk, especially during the initial sync
+process when a very large number of blocks are being processed in quick
+succession.
+
+This utxo cache provides significant runtime performance benefits at the cost of
+some additional memory usage.  The maximum size of the cache can be configured
+with the new `--utxocachemaxsize` command-line configuration option.  The
+default value is 150 MiB, the minimum value is 25 MiB, and the maximum value is
+32768 MiB (32 GiB).
+
+Some key properties of the cache are as follows:
+
+* For reads, the UTXO cache acts as a read-through cache.  All UTXO reads go
+  through the cache.  When there is a cache miss, the cache loads the missing
+  data from disk and caches it for future lookups.
```

Comment:
```suggestion
  data from the disk and caches it for future lookups.
```

Reply:
Added, although I'm fairly certain the article is optional here.